src/js/core/history.js

```diff
@@ -45,6 +45,14 @@
 					 */
 					activeState: null,
 
+					/**
+					 * Property contains starting url for tau instance
+					 * @property {Object} baseUrl
+					 * @static
+					 * @member ns.history
+					 */
+					baseUrl: null,
```

Comment:
I verified patch. It works good but only one thing referring to property name.
The "baseUrl" may be little confusing because of TAU adds `<base>` tag to DOM. Developer may thinks that this property refers to base tag. Could you change name to something like "initialUrl", "startingUrl"?

Reply:
changed baseUrl --> startURL